miqa/core/conversion/import_export_csvs.py

```diff
@@ -29,7 +29,7 @@ def validate_file_locations(input_dict, project):
                     raw_path = Path(import_path).parent.parent / raw_path
                 if not raw_path.exists():
                     raise APIException(f'Could not locate file "{raw_path}".')
-            input_dict[key] = value
+            input_dict[key] = str(raw_path)
```

Comment:
Ah ok, I was wondering why that change was made. Without this line, though, we don't get relative paths converted to absolute paths. For example, the IXI_small import csv has file paths relative to the location of the csv. Here, we use the location of the csv to convert the relative paths to absolute paths so the files can be found. 
Is there some way that we can satisfy both?

Reply:
Ah, I missed that. First thing we should do, to prevent such oversights in the future, is to add a test for the relative path case, which should fail. Then, change the logic here to support both tests passing. It might involve an extra conditional somewhere, but that's OK as long as it satisfies all the behaviors we want to support.

If you notice any other behaviors in this method that don't have associated test cases, it'd be good to add them.

Reply:
Good idea. Another good reason for us to pay attention to code coverage soon

Reply:
Changed in [1f06c2d](https://github.com/OpenImaging/miqa/pull/318/commits/1f06c2dfd89ee714109bc9566158e6088da4b9cd)